examples/src/examples/graphics/post-effects.tsx

```diff
@@ -13,6 +13,7 @@ class PostEffectsExample extends Example {
             <AssetLoader name='bokeh' type='script' url='static/scripts/posteffects/posteffect-bokeh.js' />
             <AssetLoader name='sepia' type='script' url='static/scripts/posteffects/posteffect-sepia.js' />
             <AssetLoader name='vignette' type='script' url='static/scripts/posteffects/posteffect-vignette.js' />
+            <AssetLoader name='vignette' type='script' url='static/scripts/posteffects/posteffect-ssao.js' />
```

Comment:
Incorrect name here.

Reply:
well spotted. Fixed. The name is not used in the example so it worked regardless.